web/api/web_api_v1.c

```diff
@@ -814,6 +814,9 @@ inline int web_client_api_request_v1_info(RRDHOST *host, struct web_client *w, c
     buffer_sprintf(wb, "\t\"os_version\": \"%s\",\n", (host->system_info->host_os_version) ? host->system_info->host_os_version : "");
     buffer_sprintf(wb, "\t\"os_version_id\": \"%s\",\n", (host->system_info->host_os_version_id) ? host->system_info->host_os_version_id : "");
     buffer_sprintf(wb, "\t\"os_detection\": \"%s\",\n", (host->system_info->host_os_detection) ? host->system_info->host_os_detection : "");
+    buffer_sprintf(wb, "\t\"cores_total\": \"%s\",\n", (host->system_info->host_cores) ? host->system_info->host_cores : "");
+    buffer_sprintf(wb, "\t\"cpu_freq\": \"%s\",\n", (host->system_info->host_cpu_freq) ? host->system_info->host_cpu_freq : "");
+    buffer_sprintf(wb, "\t\"ram_total\": \"%s\",\n", (host->system_info->host_ram_total) ? host->system_info->host_ram_total : "");
```

Comment:
Is there a specific reason that we expose (and store cores, ram and cpu freq) as strings? 
In general everything is exposed as strings (except the alarms object)

Reply:
Since I began to work with Netdata, this was the pattern, I believe this happens, because the protocol used to send the response is a text protocol by definition.

Reply:
Pretty certain we're using JSON, which absolutely supports integer and floating-point values.

Reply:
You are correct @Ferroin , we are using JSON that has these supports, an important question for both, before I bring possible changes. Case we decide to do this on Netdata side, this will bring more processing to the web server every time that it generates the response, should we do this?

Reply:
I just remembered why we would want these to be strings. If the script that gives this data to the core can't figure out values for these, it returns 'unknown' as a string, so we need to support passing that on.

Reply:
All right, I will keep it as text for while.